### ISSUE TITLE
Replace FilterModsTopic metadata manipulator with a more generic version

### ISSUE THREAD

**Issue by mjordan** [MarcusBarnes/mik#252]
We should convert repeated values in source metadata to be split on a delimter and generate a MODS element for each. The FilterModsTopic manipulator does this for `<subject><topic>` elements, so we should replace this with a more generic manipulator that can be applied to any element.

**Comment by mjordan** [comment]
@MarcusBarnes I will add tests and wiki entry in next day or so and ask you to do a smoke test.

**Comment by mjordan** [comment]
For the record, here are some sample .ini options (parameters are source field, XPath to MODS mapping, delimeter):

```
[MANIPULATORS]
metadatamanipulators[] = "SplitRepeatedValues|Subjects|/subject/topic|;"
metadatamanipulators[] = "SplitRepeatedValues|Creator|/name/namePart|;"
```

**Comment by mjordan** [comment]
Wiki page: https://github.com/MarcusBarnes/mik/wiki/Metadata-manipulator:-SplitRepeatedValues

**Comment by mjordan** [comment]
@MarcusBarnes test added. Please smoke test at your convenience.

**Comment by mjordan** [comment]
Just testing this now prepping to migrate a collection and it appears that for CONTENTdm toolchains, the fist parameter, the source field, must be a nickname, not a human-readable label. Makes sense. I'll update the wiki page.

**Comment by mjordan** [comment]
Wiki page updated.

**Comment by MarcusBarnes** [comment]
Addressed in pull-request https://github.com/MarcusBarnes/mik/pull/285 (merged into master with commit 172a274db2582d931ec58ec5782c48821b3c6461).
